portafly/doc/ux_specs/global_components/modal.md

```diff
@@ -71,3 +71,17 @@
           * the spinner will be replaced by the submit button
           * the input fields will be enabled again (to allow user eventually copy paste their content)
           * an inline warning message will be displayed above the form buttons -->
+
+### _'Delete confirmation'_ modal
+* Modal's header includes the name of the current API (Product or Backend) the user is trying to delete
+* Enter API name > text input field [required]
+  * Label text includes the name of the current API the user is trying to delete
+  * API name is contained into code tags for easier copy pasting
+  * Validation error:
```

Comment:
I wouldn't include any validation errors here, only the button logic.

Reply:
I proposed to add the validation onblur to avoid user getting stuck thinking they added the correct name and the disabled button is a result of system not responding. Better to be redundant than leaving the user with no hints on what is the way forward.

Reply:
It is rather simple to implement but considering it'll be copy-pasted and that there's no other input in between, the onBlur effect doesn't show up very often:
![deletemodal](https://user-images.githubusercontent.com/11672286/87636721-718ba400-c741-11ea-9980-7861b126bb83.gif)

One would think validation _onType_ would be more useful in this particular situation, being on "error" by default. This could also accentuate the danger of the action.

![deletemodal2](https://user-images.githubusercontent.com/11672286/87637761-02af4a80-c743-11ea-8652-3a9485142d40.gif)

Reply:
I also prefer `onType` rather than `onBlur`

Reply:
@josemigallas @damianpm agreed, changed!

Reply:
@josemigallas @damianpm agreed, changed!